src/app/iteration/iteration-modal/iteration-modal.component.html

```diff
@@ -22,6 +22,18 @@
           </div>
         </div>
         <div class="form-group">
+          <label for="parent-iteration">Parent Iteration</label>
+          <div class="parent-iteration-container">
+            <input id="parent-iteration" class="form-control" type="text" #iterationSearch (keyup)="filterIteration($event)"
```

Comment:
Design for picking a parent iteration:
<img width="512" alt="screen shot 2017-03-30 at 2 09 36 pm" src="https://cloud.githubusercontent.com/assets/4032718/24519262/943284d2-1552-11e7-967f-13b14ec042d7.png">